src/cljc/athens/common_events/resolver.cljc

```diff
@@ -966,3 +967,115 @@
                       {:db/id [:block/uid uid] :block/string new-str}))
                   unlinked-refs)]
     tx-data))
+
+
+(defn text-to-blocks
```

Comment:
I'm a bit confused here, I see a lot of code was added but no code was removed. But the paste functionality already existed, so shouldn't its old code be removed in this PR?

Reply:
Indeed, it wasn't removed from `events.cljs`

Reply:
Now it's gone from `events.cljs`.